packages/example/README.md

```diff
@@ -21,14 +21,31 @@ cd connect-es-main/packages/example
 npm install
 ```
 
-Next, start the Connect server:
+Next, we are going to make a locally-trusted development certificates with `mkcert`.
+If you don't have it installed yet, please run the following commands:
+
+```shell
+brew install mkcert
+mkcert -install
+mkcert localhost 127.0.0.1 ::1
+export NODE_EXTRA_CA_CERTS="$(mkcert -CAROOT)/rootCA.pem"
+```
+
+If you don't use macOS or `brew`, see the [mkcert docs](https://github.com/FiloSottile/mkcert#installation) 
+for instructions. You can copy the last line to your `~/.zprofile` or `~/.profile`, so that 
+the environment variable for Node.js is set every time to open a terminal.
+
+If you already use `mkcert`, just run `mkcert localhost 127.0.0.1 ::1` to issue a certificate 
+for our example server.
+
+Once we have that out of the way, let's start the Connect server:
```

Comment:
It's really unfortunate that no browser supports H2C (plain-text HTTP/2). If we want to show both gRPC (many clients require H2) and a web client, we can't use H2C. Previously, this was using a self-signed certificate, and you had to manually accept the connect in the web browser.

`mkcert` installs a CA that is picked up by browsers and other tools like `buf curl` and it's typically a one-time setup, so I figure it is worth using it here.